apps/experiments/views/experiment.py

```diff
@@ -1158,3 +1159,23 @@ def download_file(request, team_slug: str, session_id: int, pk: int):
         return FileResponse(file, as_attachment=True, filename=resource.file.name)
     except FileNotFoundError:
         raise Http404()
+
+
+@require_POST
+@transaction.atomic
+@login_and_team_required
+def set_default_experiment(request, team_slug: str, experiment_id: int, pk: int):
+    experiment = get_object_or_404(Experiment, working_version_id=experiment_id, version_number=pk, team=request.team)
+    Experiment.objects.exclude(version_number=pk).filter(team__slug=team_slug, working_version_id=experiment_id).update(
+        is_default_version=False, audit_action=AuditAction.AUDIT
```

Comment:
@snopoke got the auditing error without adding the "audit_action" but that feels bizarre for this instance to me-- is there a work around?

Reply:
This [is expected](https://github.com/dimagi/django-field-audit?tab=readme-ov-file#auditing-special-queryset-writes). Since the .update method [doesn't call .save](https://docs.djangoproject.com/en/5.1/ref/models/querysets/#django.db.models.query.QuerySet.update:~:text=Finally%2C%20realize%20that,%2C%20like%20this%3A), we need to manually mark this as being audited